mozfun/norm/fenix_app_info/udf.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE FUNCTION norm.fenix_app_info(app_id STRING, app_build_id STRING)
+RETURNS STRUCT<app_name STRING, channel STRING, app_id STRING> AS (
```

Comment:
Changed the `canonical_app_id` field to just `app_id` for simplicity. Particularly because the function name no longer uses "canonical".